php/utils.php

```diff
@@ -234,8 +239,8 @@ function locate_wp_config() {
 
 		if ( file_exists( ABSPATH . 'wp-config.php' ) ) {
 			$path = ABSPATH . 'wp-config.php';
-		} elseif ( file_exists( ABSPATH . '../wp-config.php' ) && ! file_exists( ABSPATH . '/../wp-settings.php' ) ) {
-			$path = ABSPATH . '../wp-config.php';
+		} elseif ( @file_exists( dirname( ABSPATH ) . '/wp-config.php' ) && ! @file_exists( dirname( ABSPATH ) . '/wp-settings.php' ) ) {
```

Comment:
Why the error silencing here? `@` silences errors in PHP, see http://php.net/manual/en/language.operators.errorcontrol.php.

Reply:
Yep, that's exactly what core is doing. 
Happy to do normal call.

Reply:
This is probably to suppress the warning that can be generated:
<img width="309" alt="image 2018-10-29 at 9 17 12 am" src="https://user-images.githubusercontent.com/83631/47637265-6fdc1000-db5b-11e8-8154-e031c6f89945.png">

We don't need to suppress these for WP-CLI, we're not generating HTML here, so I'd prefer to get rid of the error silencing.